tests/test_mtdome_csc.py

```diff
@@ -64,7 +64,7 @@ async def test_standard_state_transitions(self) -> None:
             initial_state=salobj.State.STANDBY,
             config_dir=None,
             simulation_mode=1,
-            settings_to_apply="",
+            override="",
```

Comment:
This is the default. You might consider omitting it.

Reply:
I am quite concerned about config_dir=None. I think you should change these all to point to `tests/data/config` so that you can be sure exactly what config will be used.

The risk of using the default config dir (ts_config_mttcs) is that if some of the config is site-specific (likely so) then `_init.yaml` isn't complete and if there is no `_test.yaml` your tests will fail due to incomplete configuration.

Another option is to add `_test.yaml` to `ts_config_mttcs` as long as you don't mind having somewhat harder-to-control config. But I feel result is more fragile than having the full config in your own package.

Reply:
Both done. Good suggestion to add a test `_init.yaml` config file, thanks!